resources/views/frontend.blade.php

```diff
@@ -1,48 +1,76 @@
+  There are some new classes which i added to some Tags. I needed to add those Classes so i can use them for adding the CSS styles. 
+  I did not change any ID or Classes that were writteen before i only added new Classes and tried to make their name as clear as possible.
+-->
+
 <!DOCTYPE html>
-<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-
-        <title>@lang('messages.pagetitle')</title>
-
-    </head>
-    <body>
-
-@if (session('alert'))
-    <div class="alert alert-success">
-        {{ session('alert') }}
-    </div>
-@endif
-
-<form action="{{ route('frontend.store') }}" method="post">
-  @csrf
-  <input type="hidden" name="uuid" value="{{$uuid}}"/>
-  <fieldset>
-   @lang('messages.my_name'): <input type="text" id="name" name="name" required value="{{old('name')}}"><br/>
-  </fieldset>
-  <fieldset>
-    <p>@lang('messages.select_service'):</><br/>
-    <table>
-@foreach ($services as $service)
-    <tr><td>
-    <input type="radio" id="service-{{ $service->id }}" name="service_id" value="{{$service->id}}" required>
-    <label for="service-{{ $service->id }}">{{$service->description}}</label>
-    </td><td>
-       @lang('messages.currently_visitors', ['value' => $service->count_adults + $service->count_children])
-    </td></tr>
-@endforeach
-    </table>
-  </fieldset>
-
-  <fieldset>
-    <input type="number" id="quantityAdults" name="count_adults" min="1" max="9" value="{{old('count_adults', 1)}}" required>
-    <label for="quantityAdults">@lang('messages.number_of_visitors')</label>
-    <input type="hidden" id="quantityChildren" name="count_children" value="0"/>
-  </fieldset>
-
-  <input type="submit" value="@lang('messages.submit')">
-</form>
-
-    </body>
+<html lang="de">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- I added this Link Tag in order to connect the CSS file to the HTML file.
+    Maybe the (href) needs to be changed ! because now the location of the CSS file is changed so the new location or Url should be added.-->
+    <link rel="stylesheet" href="index.css">
+
+    <title>Gottesdienst Planer</title>
+
+</head>
+
+<body>
+
+
+    <form class="form" action="https://demo.wo2odermehr.de/frontend" method="post">
+        <input type="hidden" name="_token" value="j0zFZONqzj202eK8g2tYc2ebXnQ8DYQnCYk3O4zg"> <input type="hidden" name="uuid" value="4b858c45-4921-44fa-a154-e9cd5d07e98d" />
+        <fieldset class="field">
+            <!-- Here i added this Label tag so i can apply the styles to the text also -->
+            <label class="name-input" for="name" id="name">Mein Name:</label>
+            <input class="name-input" type="text" id="name" name="name" required value=""><br/>
+        </fieldset>
+        <fieldset class="field" class="main-field">
+            <p>Bitte Gottesdienst wählen:
```

Comment:
Please can you modify this file, so that you only change the head, and add your classes to the HTML tags? 
We need to keep the strings for translation: eg. @lang('messages.select_service') 
and also the variables: eg. {{old('name')}}
and the loop: @foreach ($services as $service)